javalib/src/main/scala/java/util/SplittableRandom.scala

```diff
@@ -106,27 +108,8 @@ final class SplittableRandom private (private var seed: Long, gamma: Long) {
     seed
   }
 
-  def nextInt(): Int = mix32(nextSeed())
-
-  //def nextInt(bound: Int): Int
-
-  //def nextInt(origin: Int, bound: Int): Int
+  override def nextInt(): Int = mix32(nextSeed())
```

Comment:
Intentional not to inherit the default?

https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/util/SplittableRandom.html suggests this is not overridden.

Reply:
Yes, that's intentional. If I remove it, the existing `SplittableRandomTest` fails. I don't know how the JDK implementation produces the values it produces without overriding this method.

I'll add a comment. I should have done that in the first place.